notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -446,6 +446,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Only include datapoints whose ids are 0-100 in the dataset\n",
+    "# Add allow_list 0-100 so that only datapoints whose ids are in 0-100 are allowed to show up in the output\n",
```

Comment:
The user can specify what to allow in the query. This step is more like adding tags to the data for further filtering.

Let's update it to something like: "Add restricts to each data point, in this demo, we only add one namespace and the allowlist is set to be the same as the id. Later on, we will demo how to return only the allowlisted data points."

Reply:
Changed